dart-impl/base/src/logging.c

```diff
@@ -22,3 +22,29 @@ const int dart__base__unit_term_colors[DART_LOG_TCOL_NUM_CODES-1] = {
   DART_LOG_TCOL_RED,
   DART_LOG_TCOL_BLUE
 };
+
+
+enum dart__base__logging_loglevel
+dart__base__logging_env_loglevel()
+{
+  static enum dart__base__logging_loglevel level = DART_LOGLEVEL_TRACE;
+  static int log_level_parsed = 0;
+  if (!log_level_parsed) {
+    const char *envstr = getenv(DART_LOGLEVEL_ENVSTR);
+    if (envstr) {
```

Comment:
DART should not use `getenv` itself.
The run-time configuration of DART is maintained in `dart_config`, there already is a field `log_enabled` which is used by DASH.
The utiliy class `dash::util::Config` encapsulates the parsing of environment variables and already supports `DASH_ENABLE_LOGGING=OFF` (or =0, =1, =yes, ...) among many other environment settings (enabling tracing, chunk size settings, and the like).